src/Ampersand/Test/Parser/ArbitraryTree.hs

```diff
@@ -151,7 +161,7 @@ instance Arbitrary P_Context where
       <*> arbitrary
```

Comment:
In line 128 I see an `instance Arbitrary Origin where arbitrary = pure OriginUnknown`. I would expect some more variation in Origins.